### PR TITLE
Add support for Hotmixradio.fr

### PR DESCRIPTION
**Pull request by calvinturbo**
Just quickly wrote a connector for Hotmixradio.fr, works with all the different radiostations on their website

### CODE REVIEW

**Comment by calvinturbo** [comment]
I see there's extra space or something and that's why the check has failed. Can someone correct this since I wrote this in Notepad++? Don't have other tools available atm

File: src/connectors/hotmixradio.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Connector.playerSelector = '#new_ajax-player';
+
+Connector.artistSelector = '#hotmixPlayerSongArtist .ajax';
+
+Connector.getArtist = () => getScrollableText('#hotmixPlayerSongArtist');
+
+Connector.getTrack = () => getScrollableText('#hotmixPlayerSongTitle');
+
+//Connector.getAlbum = () => getScrollableText('#songalbum');
```
Comment (antonin-lebrard) [inline src/connectors/hotmixradio.js]: Actually you only have to add a space here, after the `//` to resolve the travis error. This should be possible with Notepad++

Comment (calvinturbo) [reply src/connectors/hotmixradio.js]: Thanks, I will just remove the line

**Comment by calvinturbo** [comment]
Have been using this script to scrobble from Hotmixradio.fr for a couple of hours ago, seems to work perfectly.

**Pull request merged**